.openpublishing.redirection.json

```
@@ -1230,10 +1230,22 @@
             "source_path": "docs/testing/sideload-an-office-add-in-on-ipad-and-mac.md",
             "redirect_url": "/office/dev/add-ins/testing/sideload-an-office-add-in-on-mac"
         },
+        {
+            "source_path": "docs/quickstarts/outlook-quickstart.md",
+            "redirect_url": "/office/dev/add-ins/quickstarts/outlook-quickstart-yo"
```

If we end up keeping the redirects, consider settinging redirect_document_id to true which I believe carries forward the data history of the old name (https://review.learn.microsoft.com/en-us/help/platform/redirect-articles?branch=main#main-redirect-file)

```suggestion
            "redirect_url": "/office/dev/add-ins/quickstarts/outlook-quickstart-yo",
            "redirect_document_id": true
```